### PR TITLE
fix edit button

### LINKED ISSUE #13: Add edit button
Inspiration: https://github.com/rust-lang/mdBook/blob/master/guide/book.toml#L14

### PR DESCRIPTION
**Pull request by michaelkirk**
note: I haven't tested this.

Fixes #13

### CODE REVIEW

File: .github/workflows/book.yml [hunk 1]
```diff
@@ -13,9 +13,7 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Install mdbook
-        uses: peaceiris/actions-mdbook@v1
-        with:
-          mdbook-version: "0.3.7"
```
Comment (michaelkirk) [inline .github/workflows/book.yml:18]: alternatively, we could do:

```
uses: peaceiris/actions-mdbook@v1
with:
    mdbook-version: "latest"
```

But, I'm not sure what we're getting from using this action vs. using cargo, it just seems like complexity overkill.

**Comment by Robinlovelace** [comment]
+1 to that, I created a commit that did a similar thing but forgot to put in the PR! https://github.com/robinlovelace/docs/tree/patch-2

File: .github/workflows/book.yml (hunk 1 above)
Comment (dabreegster) [reply .github/workflows/book.yml:18]: Ah, nice! Let's try this out, yeah. I used the other action previously to speed up deployment -- no compiling mdbook. But I guess that other action isn't keeping up to date

**Review by dabreegster — COMMENTED** [review]
Thanks!

**Pull request merged**